### PR TITLE
Sort command

### LINKED ISSUE #101: Add keydate sorting command

### LINKED ISSUE #102: Add keydate sorting parser

### LINKED ISSUE #103: Add keydate sorting feature

### PR DESCRIPTION
**Pull request by antonlee59**
close #101 
close #102 
close #103

### CODE REVIEW

File: src/main/java/seedu/ultron/logic/commands/ListCommand.java [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class ListCommand extends Command {
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
+        System.out.println(model.getFilteredOpeningList().size());
```
Comment (Lanreath) [inline src/main/java/seedu/ultron/logic/commands/ListCommand.java:21]: Remove any debugging comments

File: src/main/java/seedu/ultron/model/ModelManager.java [hunk 2]
```diff
@@ -125,7 +128,23 @@ public ObservableList<Opening> getFilteredOpeningList() {
     @Override
     public void updateFilteredOpeningList(Predicate<Opening> predicate) {
         requireNonNull(predicate);
-        filteredOpenings.setPredicate(predicate);
+        filteredOpenings.clear();
```
Comment (Lanreath) [inline src/main/java/seedu/ultron/model/ModelManager.java:131]: Does the setPredicate method not work with ContainsKeydatePredicate?

**Review by Lanreath — CHANGES_REQUESTED** [review]
A few clarifications.
Once I merge the sorted date ArrayList, DateComparator can be simplified since the smallest date will always show up first 👍

File: src/main/java/seedu/ultron/model/ModelManager.java (hunk 2 above)
Comment (antonlee59) [reply src/main/java/seedu/ultron/model/ModelManager.java:131]: I changed filteredOpenings to an Observable List instead of FilteredList

**Pull request merged**